code/defines/obj/weapon.dm

```diff
@@ -12,6 +12,49 @@
 	w_class = W_CLASS_SMALL
 	attack_verb = list("calls", "rings", "dials")
 	hitsound = 'sound/weapons/ring.ogg'
+	var/obj/landline/landline
+	
+/obj/item/weapon/phone/New()
+	landline = new /obj/landline/red (src,src)
+	var/list/a = list("alpha","bravo","charlie","delta","echo","foxtrot","golf","hotel","india")
+	var/list/b = list("anton","boris","vasilij","grigorij","dimitrij","elena","zhenja","ivan","nikolaj")
+	var/list/c = list("1","2","3","4","5","6","7","8","9")
+	name += " [pick(a)]-[pick(b)]-[pick(c)]"
+	
+/obj/item/weapon/phone/verb/pick_up_phone()
+	set category = "Object"
+	set name = "Pick up telephone"
+	set src in oview(1)
+	if(!landline)
+		to_chat(usr, "\the [src] model does not come with a telephone!")
+		return
+	//moved to landline
+	landline.pick_up_phone(usr)
+	
+/obj/item/weapon/phone/verb/dial()
+	set category = "Object"
+	set name = "Dial"
+	set src in oview(1)
+	if(!ishuman(usr))
+		to_chat(usr, "You are not capable of such fine manipulation.")
+		return
+	var/list/obj/item/weapon/phone/redphones = list()
+	for(var/obj/item/weapon/phone/P in world)
+		redphones += P
+	var/obj/item/weapon/phone/P = input("Where would you like to call?", "dialler") as null|anything in redphones
```

Comment:
typo

Reply:
dialler? valid in british

Reply:
Interesting, but what's more interesting is that if you put Br'ish English in your messages I will take you through the way out to the boot of my lorry and go on the main road to my flat for a dustbin holiday and post you to the chemist in crisps form. Do the maths.

Reply:
we have 230 files using colour and 1.1k using color, so i assumed using either form was fine